### DISCUSSION TITLE
Is it possible to add `title` and `description` for algorithm?

### DISCUSSION THREAD

**Discussion by JinIgarashi** [developmentseed/titiler#786] [Feature Request]
@vincentsarago 
I can add any parameters and metadata (under `inputs_` and `outputs_`), but current titiler algorithm cannot have `title` and `description`. These properties can be very useful for users because most of users don't know what is `terrarium`, `terrainrgb`, or custom algorithm.

It would be nice if `title` and `description` properties can be included optionally in algorithm details. Is it possible to have them in the response?

For example, response can be as below JSON.
```json
{
  "title": "Hillshade",
  "description": "Hillshade layer description",
   "inputs":{
      "nbands":1
   },
   "outputs":{
      "nbands":1,
      "dtype":"uint8",
      "min":null,
      "max":null
   },
   "parameters":{
      "azimuth":{
         "default":90,
         "title":"Azimuth",
         "type":"integer"
      },
      "angle_altitude":{
         "default":90,
         "title":"Angle Altitude",
         "type":"number"
      },
      "buffer":{
         "default":3,
         "title":"Buffer",
         "type":"integer"
      }
   }
}
```

**Comment by vincentsarago** [comment]
Sure 🙏 

feel free to add this in #783

**Reply by JinIgarashi** [reply]
@vincentsarago I added title and description in the PR#783
